src/app/core/ui/tabs/tab.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'ct-tab-component',
+  template: `
+    <div [hidden]="!active" class="pane">
+      <ng-content></ng-content>
+    </div>
+  `
+})
+export class TabComponent {
+  @Input('tabTitle') title: string;
```

Comment:
Split decorator and declaration into separate rows (as outlined in the [style guide](https://github.com/rabix/cottontail-frontend/blob/master/doc/dev/coding-style-guide.md))